ide/css.editor/external/binaries-list

```diff
@@ -15,4 +15,4 @@
 # specific language governing permissions and limitations
 # under the License.
 901D8F815922C435D985DA3814D20E34CC7622CB css21-spec.zip
-B9A045802E3214909501B822177EE3AA2F4A469F css3-spec.zip
+72202BE5A9B984CDE2B6FEE9A86DDBABA7922384 css3-spec.zip
```

Comment:
Mmm... if we upload a  new "css3-spec.zip" with a different shasum...

- Will we be breaking previous NetBeans builds? 
- Or builds in git branches that have yet merged this new checksum?